docs/develop/sdks-integrations/redis-client-compatibility.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 1
+sidebar_class_name: sidebar-item-redis
+sidebar_label: Redis compatibility client
+pagination_prev: null
+title: Redis compatibility client
+description: Quickly switch from using Redis to Momento Cache with these drop in replacement client libraries
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Redis compatibility client for Momento Cache and Momento Topics
+Do you have existing apps that use a Redis cache, but now you're trading up to Momento’s services? With Redis compatibility clients, there's no need to refactor your code. Instead, compatibility clients are a drop-in replacement for existing Redis clients. Change your client library to the compatibility client, change the connection information, and the core of your code stays the same.
+
+This client facilitates you using Momento Cache as your cache engine for any node.js frameworks that support a redis-backed cache, such as [Express.js](https://github.com/expressjs/express). See the [examples section](https://github.com/momentohq/momento-node-redis-client#examples) for more info.
+
+:::info
+
+Momento Cache and Momomento Topics do not support all of Redis' operations and datatypes. If a Redis operation is not supported by the Redis compatibility client, it will throw an error.
```

Comment:
my 2c is still that we should mention something here about our ability / willingness to add missing operations if a user needs them, but I will defer to your judgment.